File: 백준/Bronze/14720. 우유 축제/우유 축제.cc
#include <stdio.h>
main(){
	int n,cnt=0,temp=0;
	scanf("%d",&n);
	int L[n]={0};
	for(int i=0;i<n;i++){
		scanf("%d",&L[i]);
	}
	
	for(int i=0;i<n;i++){
		if (L[i]==temp){
			cnt++;
			if (temp==2)
				temp=0;
			else
				temp++;
		}
		else
			continue;
		
	}
	printf("%d",cnt);
}

File: 백준/Bronze/14720. 우유 축제/우유 축제.py
n = int(input())

L = [int(x) for x in input().split()]
cnt = 0
temp = 0
for i in L:
    if i == temp:
        cnt+=1
        if temp==2:
            temp=0
        else:
            temp += 1
    else:
        continue
print(cnt)
        
